src/Compiler/Checking/MethodOverrides.fs

```diff
@@ -380,8 +380,7 @@ module DispatchSlotChecking =
                     let compiledSig = CompiledSigOfMeth g amap m dispatchSlot
                     
                     let noimpl() =
-                        if dispatchSlot.IsInstance then
-                            missingOverloadImplementation.Add((isReqdTyInterface, lazy NicePrint.stringOfMethInfo infoReader m denv dispatchSlot))
+                        missingOverloadImplementation.Add((isReqdTyInterface, lazy NicePrint.stringOfMethInfo infoReader m denv dispatchSlot))
```

Comment:
It should probably account for object expression:
If it's object expression, then only check for instance ones
If it's just normal inheritance, then check all.

Reply:
@vzarytovskii 

We currently only show an error for: 
`Object expressions cannot implement interfaces with static abstract members or declare static members.`

But personally I think it should be more correct to also show that `static abstract member Execute: string -> unit` has not been implemented. As that is what you expect from a "type" implementing an interface ? 

```fsharp
type ILogger =
    abstract member Log: string -> unit
    static abstract member Execute: string -> unit
        
let consoleLogger =
    { new ILogger with
        member this.Log(message: string) =
            printfn "%s" message
    }
    
consoleLogger.Log("Hello World")
```

Reply:
Even if the user will implement the `static abstract member` `Error 3860` will still be there.

Reply:
What I meant is that if you just remove the condition, then if you try to use object expression with both static and instance abstract methods, you will get 

No implementation was given for 'static abstract ...'.

For it. Which is wrong, because you can't implement static abstracts in object expressions.

Reply:
Done